geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/AttributeIdxStrategy.scala

```diff
@@ -271,26 +277,29 @@ object AttributeIdxStrategy extends StrategyProvider {
       descriptor <- Option(sft.getDescriptor(attribute))
       binding    =  descriptor.getType.getBinding
       bounds     <- FilterHelper.extractAttributeBounds(f, attribute, binding)
-      if bounds.bounds.nonEmpty
     } yield {
-      // join queries are much more expensive than non-join queries
-      // TODO figure out the actual cost of each additional range...I'll make it 2
-      val additionalRangeCost = 1
-      val joinCost = 10
-      val multiplier =
-        if (descriptor.getIndexCoverage == IndexCoverage.FULL ||
-              IteratorTrigger.canUseAttrIdxValues(sft, filter.secondary, transform) ||
-              IteratorTrigger.canUseAttrKeysPlusValues(attribute, sft, filter.secondary, transform)) {
-          1
-        } else {
-          joinCost + (additionalRangeCost * (bounds.bounds.length - 1))
-        }
+      if (bounds.bounds.isEmpty) {
+        0L // disjoint range
+      } else {
+        // join queries are much more expensive than non-join queries
+        // TODO figure out the actual cost of each additional range...I'll make it 2
```

Comment:
it does seem to suggest that... I'm not entirely sure why additional ranges would make the query more expensive. @jahhulbert-ccri?